### PR TITLE
feat: courses page, single course page

### PR DESCRIPTION
**Pull request by ankerx**

### CODE REVIEW

File: src/modules/apply/components/InputsContainer.js [hunk 1]
```diff
@@ -6,8 +6,10 @@ import { LabelWithInput } from "./LabelWithInput";
 
 import { SelectComponent } from "./SelectComponent";
 
-export const InputsContainer = ({ watch, id: params }) => {
+
+export const InputsContainer = ({ watch, id }) => {
   console.log(params);
```
Comment (Nerevand) [inline src/modules/apply/components/InputsContainer.js]: remove console.log

File: src/modules/courses/Courses.js [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+  if (isLoading) return <p>Loading...</p>;
+
+  return (
+    <section className={styles.section}>
+      <h2 className={styles.heading}>Pick your perfect course!</h2>
+      <div className={styles.container}>
+        {data.map((item) => {
+          return (
+            <Course
+              id={item.id}
```
Comment (Nerevand) [inline src/modules/courses/Courses.js]: if you need all options from  your ITEM you can avoid duplicate names for key - value just if you use destructurization for your component
<Course {...item} />

File: src/modules/courses/pages/Course.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        <div className={styles.single_info_box}>
+          <Details
+            level={course.level}
+            duration={course.duration}
+            periodicity={course.periodicity}
+          />
+        </div>
+        <h5>You will learn:</h5>
+        <ul>
+          {course.description_full.map((item) => {
```
Comment (Nerevand) [inline src/modules/courses/pages/Course.js]: if you have a time you can try to add morphism(from npm)  for your APIs and rename keys that includes underscore to cammel case
description_full -> descriptionFull

File: src/modules/courses/Courses.js (hunk 2 above)
Comment (ankerx) [reply src/modules/courses/Courses.js]: yees, much better solution

**Review by Nerevand — APPROVED** [review]

File: src/modules/courses/components/Course.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    description_short: item.description_short,
+    thumbnail_img_url: item.thumbnail_img_url,
+  };
+
+  const schema = {
+    descriptionShort: "description_short",
+    thumbnailImgUrl: "thumbnail_img_url",
+  };
+
+  const { descriptionShort, thumbnailImgUrl } = morphism(schema, source);
```
Comment (Nerevand) [inline src/modules/courses/components/Course.js]: const { descriptionShort, thumbnailImgUrl } = morphism(schema, item);
  you can use like in example above. In your schema you declared what you need to rename, Other values will not be 
  available, Only these two

Comment (Nerevand) [reply src/modules/courses/components/Course.js]: don't need to created source variable

File: src/modules/courses/pages/Course.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    description_full: course.description_full,
+    full_img_url: course.full_img_url,
+  };
+
+  const schema = {
+    descriptionFull: "description_full",
+    fullImgUrl: "full_img_url",
+  };
+
+  const { descriptionFull, fullImgUrl } = morphism(schema, source);
```
Comment (Nerevand) [inline src/modules/courses/pages/Course.js]: same as in comment above. 
 const { descriptionFull, fullImgUrl } = morphism(schema, course);

**Pull request merged**